injected/src/features/web-telemetry.js

```diff
@@ -15,6 +18,10 @@ export class WebTelemetry extends ContentFeature {
         }
     }
 
+    urlChanged(navigationType = 'unknown') {
```

Comment:
Was linting moaning here about a potentially omitted arg? We don't have the ability to type this method so we probably should fix that (I've raised https://app.asana.com/1/137249556945/task/1210926450084860?focus=true follow up)